mods/cnc/audio/music.yaml

```diff
@@ -1,103 +1,63 @@
-aoi: Act On Instinct
-	VolumeModifier: 0.65
+# Base game
 airstrik: Air Strike
 	VolumeModifier: 0.9
-ccthang: C&C Thang
+j1: Untamed Land (Jungle)
 	VolumeModifier: 0.7
-ind2: Canyon Chase (Industrial 2)
+jdi_v2: Just Do It Up 2 (Take 'Em Out)
+	VolumeModifier: 0.65
+radio: Radio
 	VolumeModifier: 0.7
-heavyg: Demolition (Heavy Gear)
+rain: Rain In The Night
 	VolumeModifier: 0.7
-fwp: Fight Win Prevail
+aoi: Act On Instinct
+	VolumeModifier: 0.65
+ccthang: C&C Thang
 	VolumeModifier: 0.7
-warfare: Full Stop (Warfare)
+die: Die
 	VolumeModifier: 0.6
-win1: Great Shot!
-	Hidden: true
+fwp: Fight Win Prevail
 	VolumeModifier: 0.7
-win2: Great Shot! (SFX)
-	Filename: win1
-	Extension: var
-	Hidden: true
-iam: I Am - Times (Credits)
-	VolumeModifier: 0.6
 ind: Industrial
 	VolumeModifier: 0.65
-linefire: In The Line Of Fire
+ind2: Canyon Chase (Industrial 2)
 	VolumeModifier: 0.7
-trouble: In Trouble
-	VolumeModifier: 0.65
 justdoit: Just Do It Up
 	VolumeModifier: 0.7
-justdoit2: Just Do It Up (SFX)
-	Filename: justdoit
-	Extension: var
-	VolumeModifier: 0.7
-jdi_v2: Just Do It Up 2 (Take 'Em Out)
-	VolumeModifier: 0.65
-map1: Map Theme
-	Hidden: true
+linefire: In The Line Of Fire
 	VolumeModifier: 0.7
 march: March To Doom
 	VolumeModifier: 0.7
-nod_map1: Nod Map Theme
-	Hidden: true
-nod_win1: Nod Win Theme
-	Hidden: true
-	VolumeModifier: 0.9
-nomercy: No Mercy
+target: Target (Mechanical Man)
 	VolumeModifier: 0.7
-nomercy2: No Mercy (SFX)
-	Filename: nomercy
-	Extension: var
+nomercy: No Mercy
 	VolumeModifier: 0.7
 otp: On The Prowl
 	VolumeModifier: 0.7
-outtakes: Outtakes (Censored)
-	Hidden: true
-	VolumeModifier: 0.65
 prp: Prepare For Battle
 	VolumeModifier: 0.6
-radio: Radio
-	VolumeModifier: 0.7
-rain: Rain In The Night
-	VolumeModifier: 0.7
-target: Target (Mechanical Man)
-	VolumeModifier: 0.7
-j1: Untamed Land (Jungle)
-	VolumeModifier: 0.7
-valkyrie: Ride Of The Valkyries
-	Hidden: true
-stopthem: We Will Stop Them (Deception)
-	VolumeModifier: 0.7
-deception: We Will Stop Them (SFX)
-	Filename: stopthem
+rout: Reaching Out
 	Extension: var
-	VolumeModifier: 0.7
-aoi2: Act On Instinct (SFX)
-	Filename: aoi
+heart: Heartbreak
```

Comment:
Heartbreak is an SFX track, shouldn't it be among them?

Reply:
Same applies to Reaching Out.

Reply:
Will fix when we agreed on an order, see my comment.